build_tools/azure/posix-32.yml

```diff
@@ -0,0 +1,55 @@
+parameters:
+  name: ''
+  vmImage: ''
+  matrix: []
+
+jobs:
+- job: ${{ parameters.name }}
+  pool:
+    vmImage: ${{ parameters.vmImage }}
+  variables:
+    TEST_DIR: '$(Agent.WorkFolder)/tmp_folder'
+    JUNITXML: 'test-data.xml'
+    OMP_NUM_THREADS: '4'
+    OPENBLAS_NUM_THREADS: '4'
+    SKLEARN_SKIP_NETWORK_TESTS: '1'
+  strategy:
+    matrix:
+      ${{ insert }}: ${{ parameters.matrix }}
+
+  steps:
+    - script: >
+        docker container run --rm
+        -v $TEST_DIR:/temp_dir
+        -v $PWD:/io
+        -w /io
+        -d
+        --name skcontainer
+        -e DISTRIB=ubuntu-32
+        -e TEST_DIR=/temp_dir
+        -e JUNITXML=$JUNITXML
+        -e VIRTUALENV=testvenv
+        -e JOBLIB_VERSION=$JOBLIB_VERSION
+        -e SKLEARN_NO_OPENMP=$SKLEARN_NO_OPENMP
+        -e OMP_NUM_THREADS=$OMP_NUM_THREADS
+        -e OPENBLAS_NUM_THREADS=$OPENBLAS_NUM_THREADS
+        -e SKLEARN_SKIP_NETWORK_TESTS=$SKLEARN_SKIP_NETWORK_TESTS
+        i386/ubuntu:16.04
+        sleep 1000000
+      displayName: 'Start up'
+    - script: >
+        docker exec skcontainer ./build_tools/azure/install.sh
+      displayName: 'Install'
+    - script: >
+        docker exec skcontainer ./build_tools/azure/test_script.sh
+      displayName: 'Test Library'
+    - task: PublishTestResults@2
+      inputs:
+        testResultsFiles: '$(TEST_DIR)/$(JUNITXML)'
```

Comment:
I'm confused about how this accesses the docker container

Reply:
The `$TEST_DIR` is mapped into to the containers `/temp_dir`, so the junit xml generated by pytest will be place correctly into `$TEST_DIR` for the host to see.

Reply:
Ah, I missed the `-v`